requirements.txt

```diff
@@ -15,8 +15,9 @@ networkx==1.11
 SQLAlchemy==1.3.1
 icalendar==3.9.1
 GitPython==1.0.1
-gunicorn==19.7.1
-gevent==1.2.2
+#gunicorn==19.7.1
```

Comment:
If we're not using this, can you remove it?

Reply:
☑️